core/proto/keytransparency_v1_types/gen.go

```diff
@@ -12,6 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:generate protoc -I=. -I=$GOPATH/src/ -I=$GOPATH/src/github.com/google/trillian/ --go_out=:. keytransparency_v1_types.proto
+//go:generate protoc -I=. -I=$GOPATH/src/ -I=$GOPATH/src/github.com/google/trillian/ -I=$GOPATH/src/github.com/googleapis/googleapis --go_out=:. keytransparency_v1_types.proto
```

Comment:
hmm, Good question. I didn't notice that.. 
Hmm, I'm thinking no, because grpc-gateway isn't actually vendoring it (ie. it's not in the vendor directory), so the files that the compiler will actually be using are the native ones. 
What do you think?